proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -392,7 +392,7 @@ message MapEntry {
 
 // FileExtension provides the information for the documentation of a file extension.
 message FileExtension {
-  // extension_type is the string representation of the type being extended.
```

Comment:
Here it should be `extended` as well. I think these are the only places.

Reply:
fixed